docs/archived docs/how-to-process-language-models-with-CI-CD.md

```diff
@@ -0,0 +1,86 @@
+# Using botframework-cli tool to process language models for Composer bots
+
+Composer allows users to enable kinds of languages models such as [LUIS](https://www.luis.ai/home) and [QnAMaker](https://www.qnamaker.ai/). Composer can help to preprocess and publish these models to azure services if users provide the resource key and other essential elements. However, outside Composer, users can also use botframework-cli tool to achieve the same goal.
+
+## Prerequisites
+Botframework-cli tool is based on the Node.js platform and the [OClif](https://github.com/oclif/oclif) framework where it inherits its command line parsing style, and plugin architecture platform.
```

Comment:
List the requirement {YOUR_LUIS_AUTHORING_KEY} {YOUR_QnAMaker_SUBSCRIPTION_KEY} and  here

Reply:
Listed here.